src/main/java/de/muenchen/isi/api/controller/AbfrageStatusController.java

```diff
@@ -32,16 +32,155 @@ public class AbfrageStatusController {
+    })
+    @PreAuthorize("hasAuthority(T(de.muenchen.isi.security.AuthoritiesEnum).ISI_BACKEND_ABBRECHEN_ABFRAGE.name())")
+    public ResponseEntity<Void> abbrechenInfrastrukturabfrage(@PathVariable @NotNull final UUID id) throws EntityNotFoundException, AbfrageStatusNotAllowedException {
+        this.abfrageStatusService.abbrechenAbfrage(id);
+        return ResponseEntity.ok().build();
+    }
+
+    @PutMapping("infrastruktur-abfrage/{id}/angabe-anpassen")
+    @Transactional
+    @Operation(summary = "Setzt eine Infrastrukturabfrage auf den Status ANGELEGT")
+    @ApiResponses(value = {
+            @ApiResponse(responseCode = "200", description = "OK -> Abfrage wurde erfolgreich zurückgegeben an den Abfrage Ersteller."),
+            @ApiResponse(responseCode = "404", description = "NOT_FOUND -> Es gibt keine Abfrage mit der ID.", content = @Content(schema = @Schema(implementation = InformationResponseDto.class))),
+            @ApiResponse(responseCode = "409", description = "CONFLICT -> Die Abfrage konnte aufgrund des aktuellen Status nicht zur Bearbeitung zurückgegeben werden.", content = @Content(schema = @Schema(implementation = InformationResponseDto.class)))
+    })
+    @PreAuthorize("hasAuthority(T(de.muenchen.isi.security.AuthoritiesEnum).ISI_BACKEND_ANGABEN_ANPASSEN_ABFRAGE.name())")
+    public ResponseEntity<Void> angabenAnpassenInfrastrukturabfrage(@PathVariable @NotNull final UUID id) throws EntityNotFoundException, AbfrageStatusNotAllowedException {
+        this.abfrageStatusService.angabenAnpassenAbfrage(id);
+        return ResponseEntity.ok().build();
+    }
+
+    @PutMapping("infrastruktur-abfrage/{id}/weitere-abfragevarianten-anlegen")
+    @Transactional
+    @Operation(summary = "Setzt eine Infrastrukturabfrage auf den Status IN_ERFASSUNG")
+    @ApiResponses(value = {
+            @ApiResponse(responseCode = "200", description = "OK -> Abfrage wurde erfolgreich an Sachbearbeitung Kita/Schule zurückgegeben."),
+            @ApiResponse(responseCode = "404", description = "NOT_FOUND -> Es gibt keine Abfrage mit der ID.", content = @Content(schema = @Schema(implementation = InformationResponseDto.class))),
+            @ApiResponse(responseCode = "409", description = "CONFLICT -> Die Abfrage konnte aufgrund des aktuellen Status nicht zur Bearbeitung zurückgegeben werden.", content = @Content(schema = @Schema(implementation = InformationResponseDto.class)))
+    })
+    @PreAuthorize("hasAuthority(T(de.muenchen.isi.security.AuthoritiesEnum).ISI_BACKEND_WEITERE_ABFRAGEVARIANTEN_ANLEGEN_ABFRAGE.name())")
+    public ResponseEntity<Void> weitereAbfragevariantenAnlegen(@PathVariable @NotNull final UUID id) throws EntityNotFoundException, AbfrageStatusNotAllowedException {
+        this.abfrageStatusService.weitereAbfragevariantenAnlegen(id);
+        return ResponseEntity.ok().build();
+    }
+
+    @PutMapping("infrastruktur-abfrage/{id}/keine-zusaetzliche-abfragevariante")
```

Comment:
Kann man nicht über die Klasse die Annotation schreiben 
`@RequestMapping(value = CONTROLLER_PATH)` um doppelten Code und Tippfehler zu vermeiden.

Als Konstante setzen:
`public static final String CONTROLLER_PATH = "infrastruktur-abfrage";`

```suggestion
    @PutMapping("/{id}/keine-zusaetzliche-abfragevariante")
```